duckdb_engine/__init__.py

```diff
@@ -601,6 +605,39 @@ def get_multi_columns(
 
         return columns.items()
 
+    # fix for https://github.com/Mause/duckdb_engine/issues/1128
+    # (Overrides sqlalchemy method)
+    def _comment_query(  # type: ignore[no-untyped-def]
```

Comment:
in [the upstream code](https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5519/3/lib/sqlalchemy/dialects/postgresql/base.py#4677) they have a cache on this call. Can you add that here too? IDK if it's really needed, but it sure is easy:

```
import functools

...

@functools.cache()
def _comment_query...
```

Reply:
Fixed in the latest commit. I added lru_cache since that's what is used in the upstream code and cache was giving me errors but I did read online that they're similar functions. Is that alright?

Reply:
Yes, without a cache size I think they are the same